guides/e2b_autogen/e2b_autogen/main.py

```diff
@@ -0,0 +1,196 @@
+import json
+import logging
+
+from hashlib import md5
+from typing import Optional
+
+# Import E2B
+from e2b import Sandbox
+
+# Import autogen
+from autogen import AssistantAgent, UserProxyAgent, config_list_from_json
+
+# load .env
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Get your Sandbox session
+sandbox = Sandbox(template="base")
+
+logger = logging.getLogger(__name__)
+
+
+# mock autogen execute_code function with e2b equivalent
+# To run locally: from autogen.code_utils import execute_code
+def execute_code(code: Optional[str] = None,
+                 timeout: Optional[int] = None,
+                 filename: Optional[str] = None,
+                 work_dir: Optional[str] = "/home/user",  # default to e2b default cwd
+                 lang: Optional[str] = "python",
```

Comment:
You are not using these, I would get rid of them and keep it simple

Reply:
Sure! Can totally remove and just keep to what I'm using. I've been trying to keep the signature in line with the Autogen builtin execute_code function so it can be a drop in replacement. Let me know if that's a good direction in general or if I should just keep things simple

Reply:
I would keep it simple as possible. Taking into the account the people will use it as a reference and the simpler the easier to comprehend :))

Reply:
Okay sounds good! I'll clean this up with the focus on simplicity for this PR and can potentially revisit a more complete implementation when contributing to autogen